src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
+  const sum = days * 40;
   // write code here
+
+  if (days >= 7) {
+    return (sum - 50);
+  } else if (days < 3) {
```

Comment:
```suggestion
  if (days >= 7) {
    return (sum - 50);
  }
  
  if (days < 3) {
```
если внутри есть return - не надо писать else